src/views/MainPage/MainPage.js

```diff
@@ -84,41 +85,60 @@ function renderBubblesContent() {
 }
 
 async function renderFactBubbleContent() {
-  const factObj = await getAnimalFact();
-  renderAnimalFact(factObj);
-  renderAnimalFactImg(factObj);
+  const imageFact = await getAnimalImageFact();
+  const imageObject = new Image();
+  imageObject.addEventListener(
+    'load',
+    () => {
+      renderAnimalFact(imageFact);
+    },
+    false,
+  );
```

Comment:
There are 2 event listeners for this one object. Additionally, they are in two separate functions so it is hard to maintain. Move everything to the one handler.